### ISSUE TITLE
Training settings for SD1.4: Discrepancy with paper?

### ISSUE THREAD

**Issue by jaylee2000** [XueZeyue/DanceGRPO#58]
Hello, thank you for sharing your code.
I would appreciate clarification on the hyperparameters used for the Stable Diffusion experiments.

According to Table 6 in Appendix A of the paper, the hyperparameters are:
* lr = 1e-5
* Gradient clip norm = 1.0
* _Images per iteration = (Prompts per iteration ) X (Images per prompt) = 32 X 12 = 384_
* _Gradient updates per iteration = 1_  ("When enabling CFG, ...")
* Clip range = 1e-4
* _Timestep selection = 0.6_
(I assume Noise level = 0.3 is only for the Rectified Flow Transformer based models)

But when I view `fastvideo/config_sd` in the repo, the hyperparameters are:
* lr = 1e-5
* Gradient clip norm = 1.0
* _Images per iteration = (Prompts per iteration) X (Images per prompt) = 32 X 16 = 512_
* _Gradient updates per iteration = 2_
* Clip range = 1e-4
* _Timestep selection = 1.0_

Which configuration did you use to produce Figure 1(a) of the paper?

**Comment by XueZeyue** [comment]
Hi, the setting in our paper is for FLUX and HunyuanVideo-T2I.

You can use the settings in the config to reproduce the training curves of SD.

<img width="7359" height="4798" alt="Image" src="https://github.com/user-attachments/assets/d6b3ef03-277b-4d2c-80f1-4865285a621c" />

Multi-reward training settings can also be found [here](https://github.com/XueZeyue/DanceGRPO/issues/19)
